Cubical/Data/Sigma/Properties.agda

```diff
@@ -193,3 +196,99 @@ PiΣ = isoToEquiv (iso (λ f → fst ∘ f , snd ∘ f)
 
 swapΣEquiv : ∀ {ℓ'} (A : Type ℓ) (B : Type ℓ') → A × B ≃ B × A
 swapΣEquiv A B = isoToEquiv (iso (λ x → x .snd , x .fst) (λ z → z .snd , z .fst) (\ _ → refl) (\ _ → refl))
+
+Σ-split-iso : ∀ {ℓ} {A : Set ℓ} {B : A → Set ℓ} {a a' : A} {b : B a} {b' : B a'} → Iso (Σ (a ≡ a') (λ q → PathP (λ i → B (q i)) b b')) ((a , b) ≡ (a' , b'))
```

Comment:
Please state this Iso above https://github.com/agda/cubical/blob/master/Cubical/Data/Sigma/Properties.agda#L44 and use it in the proof of Σ≡

I think we should get better at stating the Iso, Equiv and Path versions of things

Reply:
Should I state versions for both Iso, Equiv and Path for the things I add? Currently I only have Iso and Path.